consensus/src/threshold_simplex/mocks/conflicter.rs

```diff
@@ -68,36 +68,32 @@ impl<
             // Process message
             match msg {
                 Voter::Notarize(notarize) => {
-                    // Notarize received digest
+                    // Notarize random digest
```

Comment:
Generally speaking, sending random first should cause more issues (as the first message is the one that ends up getting persisted).